modules/components/src/DataTable/TableToolbar/saveTSV.js

```diff
@@ -59,22 +59,24 @@ function getRows(args) {
     entities = [],
   } = args;
   if (pathIndex >= paths.length - 1) {
-    return columns.map(column => {
-      const entity = entities
-        .slice()
-        .reverse()
-        .find(entity => column.field.indexOf(entity.field) === 0);
+    return [
+      columns.map(column => {
```

Comment:
returning to nowhere?

Reply:
or sorry, 2d array? i guess that makes sense 💤

Reply:
yep, normalizing the return so I cal always call flatten on the return value of this function :)